src/style.css

```
@@ -0,0 +1,40 @@
+* {
+  margin: 0;
+}
+    
+ul {
+  margin: 0;
+  padding: 0;
+}
+    
+.stars {
+  display: flex;
+  list-style: none;
+  margin-right: 0;
+}
+    
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  
+  background-image: url(images/star.svg);
```

will be better to wrap with quoter the url for the image 
```suggestion
  background-image: url("images/star.svg");
```